TAO/orbsvcs/ImplRepo_Service/tao_imr_i.cpp

```diff
@@ -29,6 +29,8 @@ TAO_IMR_i::~TAO_IMR_i (void)
 int
 TAO_IMR_i::run ()
 {
+ACE_Time_Value tv(0, 100000);
```

Comment:
No, that was part of my debugging effort that I missed on cleanup. I'll wait to see some scoreboard results then do another commit late today.

> On Apr 12, 2016, at 1:57 AM, Johnny Willemsen <notifications@github.com> wrote:
> 
> In TAO/orbsvcs/ImplRepo_Service/tao_imr_i.cpp <https://github.com/DOCGroup/ATCD/pull/231#discussion_r59328445>:
> 
> > @@ -29,6 +29,8 @@ TAO_IMR_i::~TAO_IMR_i (void)
> >  int
> >  TAO_IMR_i::run ()
> >  {
> > +ACE_Time_Value tv(0, 100000);
> Is this intented???
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub <https://github.com/DOCGroup/ATCD/pull/231/files/980c5a53d91ef91ecb88d46df8eeb08e6cc0b322#r59328445>

--
Phil Mesnier
Principal Engineer & Partner

OCI | WE ARE SOFTWARE ENGINEERS.
tel  +1.314.579.0066 x225
ociweb.com <http://ociweb.com/>